diff --git a/tensorflow/lite/tools/cmake/modules/ml_dtypes/CMakeLists.txt b/tensorflow/lite/tools/cmake/modules/ml_dtypes/CMakeLists.txt
index 91e893ee377..17bcff42b95 100644
--- a/tensorflow/lite/tools/cmake/modules/ml_dtypes/CMakeLists.txt
+++ b/tensorflow/lite/tools/cmake/modules/ml_dtypes/CMakeLists.txt
@@ -24,8 +24,8 @@ endif()
 
 add_library(ml_dtypes INTERFACE)
 target_include_directories(ml_dtypes INTERFACE
-  "${ML_DTYPES_SOURCE_DIR}"
-  "${ML_DTYPES_SOURCE_DIR}/ml_dtypes")
+  "$<BUILD_INTERFACE:${ML_DTYPES_SOURCE_DIR}>" "$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>"
+  "$<BUILD_INTERFACE:${ML_DTYPES_SOURCE_DIR}/ml_dtypes>" "$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/ml_dtypes>")
 file(GLOB ML_DTYPES_PUBLIC_HEADERS
   ${ML_DTYPES_SOURCE_DIR}/ml_dtypes/include/*.h)
 set_target_properties(ml_dtypes PROPERTIES
